lib/galaxy/workflow/modules.py

```diff
@@ -1145,6 +1145,8 @@ def callback( input, value, prefixed_name, prefixed_label ):
                         if isinstance( input, DataToolParameter ):
                             # Pull out dataset instance from element.
                             replacement = iteration_elements[ prefixed_name ].dataset_instance
```

Comment:
Hmm, that gives me `<object object at 0x105f39d50>` for the replacement and then crashes.


> If you try that and it doesn't work - this is probably fine for now, but can you add a comment with a TODO comment to populate the the wrappr with the element identifier as an argument and not by modifying the HDA.

I guess we could also write directly to params_combination. I saw that this is consumed later ... I'll give that a try.